FlashCap.Core/Devices/DirectShowDevice.cs

```diff
@@ -312,6 +324,106 @@ protected override Task OnStopAsync(CancellationToken ct)
         }
     }
 
+    public override int GetPropertyValue(VideoProcessingAmplifierProperty property)
+    {
+        if (Properties.TryGetValue(property, out CaptureDeviceProperty _)
+            && graphBuilder != null
+            && captureGraphBuilder != null)
+        {
+            graphBuilder.FindFilterByName("Capture source", out NativeMethods_DirectShow.IBaseFilter? captureSourceFilter);
+            if (captureSourceFilter != null)
+            {
+                captureGraphBuilder.FindInterface(Guid.Empty, Guid.Empty, captureSourceFilter, NativeMethods_DirectShow.IAMVideoProcAmpHelper.GUID, out object? videoProcAmpObject);
+                if (videoProcAmpObject != null)
+                {
+                    var videoProcAmp = (NativeMethods_DirectShow.IAMVideoProcAmp)videoProcAmpObject;
+                    videoProcAmp.Get(DirectShowProperty.FromVideoProcessingAmplifierProperty(property), out int value, out NativeMethods_DirectShow.VideoProcAmpFlags _);
```

Comment:
I don't really the use of this one yet, `NativeMethods_DirectShow.VideoProcAmpFlags`

Reply:
Even in the environment at hand, I was able to obtain a range of values that we believe to be brightness. Nice work!